equinix/helpers_device.go

```diff
@@ -216,34 +194,6 @@ func waitForDeviceAttribute(d *schema.ResourceData, targets []string, pending []
 	return "", err
 }
 
-// powerOnAndWait Powers on the device and waits for it to be active.
-func powerOnAndWait(d *schema.ResourceData, meta interface{}) error {
```

Comment:
Just calling out that this is being removed because it's not used anywhere.  Nice find!